data/sql/updates/pending_db_world/rev_1617265345631987700.sql

```diff
@@ -0,0 +1,37 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1617265345631987700');
+
+DELETE FROM `creature_addon` WHERE (`guid` IN (17090));
+
+DELETE FROM `waypoints` WHERE (`entry` IN (3984));
+INSERT INTO `waypoints` VALUES
+(3984, 1, 888.1400, -121.8399, 38.3880, 'Nancy Vishas'),
+(3984, 2, 891.5708, -114.1264, 38.2004, 'Nancy Vishas'),
+(3984, 3, 909.1765, -90.91162, 33.7845, 'Nancy Vishas'),
+(3984, 4, 891.5708, -114.1264, 38.2004, 'Nancy Vishas'),
+(3984, 5, 888.1400, -121.8399, 38.3880, 'Nancy Vishas');
+
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 398400) AND (`source_type` = 9)
```

Comment:
Btw it is failing here due to missing colon `;`

Reply:
looooooooooooool i didn't noticed it... ty @Kitzunu

Reply:
it gave me this error before:
```ERROR 1064 (42000) at line 28586: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE (`entry` IN (3984, 398' at line 3```

Reply:
because this error i made that change..